app/view/window/text/TextView.js

```diff
@@ -126,12 +126,14 @@ Ext.define('EdiromOnline.view.window.text.TextView', {
             });
 
             return;
-        }
+        }   
```

Comment:
This looks like some debugging/testing relicts. If this is safe please remove the code fragments, otherwise please add more comments